hbp_knowledge/huntington/jiang2012.bel

```diff
@@ -0,0 +1,273 @@
+ DEFINE NAMESPACE PUBCHEM        AS PATTERN "^\d+$"
+
+###############
+# Annotations #
+###############
+
+# Annotations defined with regular expressions
+# --------------------------------------------
+DEFINE ANNOTATION Species       AS PATTERN "^\d+$"
+
+# Annotations hosted on https://arty.scai.fraunhofer.de/artifactory/bel/annotation
+# --------------------------------------------------------------------------------
+ DEFINE ANNOTATION Anatomy       AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/anatomy/anatomy-20170511.belanno"
+ DEFINE ANNOTATION CellLine      AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/cell-line/cell-line-20170511.belanno"
+ DEFINE ANNOTATION CellStructure AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/cell-structure/cell-structure-20170511.belanno"
+ DEFINE ANNOTATION Cell          AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/cell/cell-20170511.belanno"
+ DEFINE ANNOTATION Confidence    AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/confidence/confidence-1.0.0.belanno"
+ DEFINE ANNOTATION Disease       AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/disease/disease-20170511.belanno"
+ DEFINE ANNOTATION Gender        AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/gender/gender-1.0.0.belanno"
+ DEFINE ANNOTATION MeSHAnatomy   AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/mesh-anatomy/mesh-anatomy-20170511.belanno"
+ DEFINE ANNOTATION MeSHDisease   AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/mesh-diseases/mesh-diseases-20170511.belanno"
+ DEFINE ANNOTATION Subgraph      AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/neurommsig/neurommsig-1.0.3.belanno"
+ DEFINE ANNOTATION TextLocation  AS URL "https://arty.scai.fraunhofer.de/artifactory/bel/annotation/text-location/text-location-1.0.1.belanno"
+
+##############
+# Statements #
+##############
+SET Citation = {"PubMed", "22179319"}
+
+SET Evidence = "SIRT1 is one of seven identified sirtuins, an evolutionarily conserved proteins with NAD+-dependent deacetylase activity that participate in numerous cell activities including cellular metabolism[5]."
+p(HGNC:"SIRT1") -- path(MESH:"Metabolism")
+p(HGNC:"SIRT1") isA p(HGNCGENEFAMILY:"Sirtuins")
+
+SET Evidence = "Whether SIRT1 has protective role in mammalian HD remains undefined, however, as results from lower organisms such as Caenorhabditis elegans [13] and Drosophila melanogaster [14] are contradictory; for example, overexpression of sir2, an ortholog of human SIRT1, suppressed mutant huntingtin (Htt)-induced neurotoxicity [13], whereas overexpression of sir2 in drosophila showed no significant protection [15]."
+#34708 - Sirt1 D.melanogaster
+#3361342 - sir2 C.elegans
```

Comment:
There's specifically the flybase namespace (FB) for this. Try and use that next time @YojanaGadiya @trushaadeshara8 @sgebel